src/coreclr/vm/i386/asmhelpers.asm

```diff
@@ -1235,6 +1236,49 @@ _GenericCLRToCOMCallStub@0 proc public
     ret
 
 _GenericCLRToCOMCallStub@0 endp
+
+_GenericComCallStub@0 proc public
```

Comment:
This ~20 instruction asm stub is a replacement for the several hundred lines of stub linker code that I am deleting. I have verified that this path is executed by (multiple) tests.